assets/scss/repeat.scss

```diff
@@ -1097,5 +1173,10 @@ button.show-date-filter {
 }
 
 .modal-content {
-  border-radius: var(--wsBorderRadius, unset);
+  border-radius: var(--schedules-border-radius, unset);
+}
+
+// We must undo a Font Family used in Canadian Colorway Typography for icons only.
+span.material-symbols-outlined {
+  font-family: 'Material Symbols Outlined', serif !important;
```

Comment:
Why do we need !important here?

Reply:
@podarok unfortunately,  it doesn't work without ` !important`
![image](https://github.com/user-attachments/assets/8800c86b-2e46-4d53-afce-baa5da1329d7)
As alternative, we can add a class `material-symbols-outlined `to this place
https://git.drupalcode.org/project/ws_colorway_canada/-/blob/1.2.x/assets/scss/canada-typography.scss?ref_type=heads#L26
Will this solution be preferable for you?
cc @rollki